featuretools/primitives/rolling_primitive_utils.py

```diff
@@ -153,24 +140,106 @@ def _check_window_size(window_size):
         raise TypeError("Window length must be either an offset string or an integer.")
 
 
-def _check_gap(window_size, gap):
+def _check_gap(window_size: Union[int, str], gap: Union[int, str]) -> None:
     # Gap must either be a valid offset string that also has an offset string window length
     if isinstance(gap, str):
         if not isinstance(window_size, str):
             raise TypeError(
                 f"Cannot roll series with offset gap, {gap}, and numeric window length, {window_size}. "
                 "If an offset alias is used for gap, the window length must also be defined as an offset alias. "
-                "Please either change gap to be numeric or change window length to be an offset alias."
+                "Please either change gap to be numeric or change window length to be an offset alias.",
             )
         try:
             to_offset(gap)
         except ValueError:
             raise ValueError(
-                f"Cannot roll series. The specified gap, {gap}, is not a valid offset alias."
+                f"Cannot roll series. The specified gap, {gap}, is not a valid offset alias.",
             )
     # Or an integer greater than or equal to zero
     elif isinstance(gap, int):
         if gap < 0:
             raise ValueError("Gap must be greater than or equal to zero.")
     else:
         raise TypeError("Gap must be either an offset string or an integer.")
+
+
+def apply_rolling_agg_to_series(
+    series: Series,
+    agg_func: Callable[[Series], float],
+    window_size: Union[int, str],
+    gap: Union[int, str] = 0,
+    min_periods: int = 1,
+    ignore_window_nans: bool = False,
+) -> np.ndarray:
+    """Applies a given aggregation function to a rolled series.
+
+    Args:
+        series (Series): The series over which rolling windows will be created. The series must have a numeric column
```

Comment:
Idk that a Series can be described as containing a numeric column since it kind of is just a column? Maybe something like  "The series must have numeric values and a DatetimeIndex." Feel free to ignore this if it's overly pedantic.

Reply:
Makes sense to me I'll change it.